lib/get-message.js

```diff
@@ -1,11 +1,30 @@
-const _ = require('lodash')
+const { defaultCommitMessages } = require('./default-commit-messages')
 
 module.exports = function (commitMessages, key, values) {
-  if (!Object.prototype.hasOwnProperty.call(commitMessages, key)) {
+  if (!Object.prototype.hasOwnProperty.call(commitMessages, key) &&
+  !Object.prototype.hasOwnProperty.call(defaultCommitMessages, key)) {
```

Comment:
This line shouldn't be necessary unless `get-message` is being called with a custom config object, under regular usage in the code the object should be the result of a call to `getConfig` which will have used the `default-commit-messages` as the base in the first place, so this second condition could never trigger.

Reply:
(It might be nice to add a comment about that?)

Reply:
We were seeing the error when someone did not define all the custom commit messages. Nothing bad happened, the default message was used but the `Unknown message messageKey` error showed up in our Rollbar. That is just my quick fix for that. I'm not sure why commitMessages somehow is not the combined object of default and user defined ones...

Reply:
@Arcanemagus  found out why: https://github.com/greenkeeperio/greenkeeper/pull/620 !